srcpkgs/ivpn/template

```diff
@@ -0,0 +1,65 @@
+# Template file for 'ivpn'
+pkgname=ivpn
+version=3.4.5
+revision=1
+_ivpndir="desktop-app-${version}"
+archs="x86_64"
+create_wrksrc=yes
+hostmakedepends="curl w3m libcurl git go wireless_tools-devel"
+depends="openvpn wireless_tools glibc lsof"
+short_desc="IVPN Command Line Interface"
+maintainer="b-l-a-i-n-e <blaine.gilbreth@gmail.com>"
+license="custom:Proprietary"
+homepage="https://www.ivpn.net"
+distfiles="https://github.com/ivpn/desktop-app/archive/v${version}.tar.gz"
+checksum=e6837cb653dc9089afa50281bc11df5da99da0b6edd597220f50646db0c8cccd
+_license_checksum=92fd1c283e604a160a9dd3ef3d030fae302bc49c6f8c62c27e3dff358ecb2d9d
+repository=nonfree
+restricted=yes
+ostrip=yes
+nopie=yes
+
+post_extract() {
+	curl -L https://raw.githubusercontent.com/ivpn/ivpn.net/main/src/content/pages/terms-of-service.md > EULA
```

Comment:
This feels a little out of place for me since it's a EULA of the IVPN service rather than the software. As maintainers, we're only providing the software and in my eyes any acceptance of a service agreement is between the service provider and the user.

If you agree, this doesn't need to be in the nonfree repo since the build scripts seem to be pulling in all open source software and the licence should be changed to GPL3-only.

Reply:
@NeelChotai makes sense to me, I'll remove those eula checks and change to GPL3 license